swagger.yaml

```diff
@@ -236,15 +266,39 @@ definitions:
       name:
         type: "string"
         example: "Dan's Trail"
+      zipCode:
+        description: "Zip code of trail head"
+        type: "integer"
       difficulty:
         type: "string"
         enum:
         - "Green"
         - "Blue"
         - "Black"
         - "Double Black"
-      features:
+      attributes:
         $ref: "#/definitions/Features"
-      location:
-        description: "Zip code of trail head"
-        type: "integer"
\ No newline at end of file
+
+  Trails:
+    type: "object"
+    properties:
+      data:
+        type: "array"
+        items:
+          type: "object"
+          properties:
+            id:
+              type: "integer"
+              format: "int64"
+            type:
+              type: "string"
+            attributes:
+              $ref: "#/definitions/Trail"
+
+  Error:
```

Comment:
Let's use the same error object that the skeleton uses, since the skeleton will return this structure by default: https://github.com/osu-mist/web-api-skeleton/blob/a3370d91e65fd4bb1a3c6ed48460a15c8f3eea0c/swagger.yaml#L59-L75